code/stable/dblpend/SRS/Markdown/DblPend_SRS.md

```diff
@@ -0,0 +1,1128 @@
+
+<div id="twoDMotion"></div>
+
+twoDMotion: The pendulum motion is two-dimensional (2D).
+
+<div id="cartSys"></div>
+
+cartSys: A Cartesian coordinate system is used.
+
+<div id="cartSysR"></div>
+
+cartSysR: The Cartesian coordinate system is right-handed where positive \\(x\\)-axis and \\(y\\)-axis point right up.
+
+<div id="yAxisDir"></div>
+
+yAxisDir: The direction of the \\(y\\)-axis is directed opposite to gravity.
+
+<div id="startOrigin"></div>
+
+startOrigin: The first rod is attached to the origin.
+
+<div id="firstPend"></div>
+
+firstPend: The first rod has two sides. One side attaches to the origin. Another side attaches to the first object.
+
+<div id="secondPend"></div>
+
+secondPend: The second rod has two sides. One side attaches to the first object. Another side attaches to the second object.
+
+
+# Theoretical Models {#Sec:TMs}
+
+This section focuses on the general equations and laws that DblPend is based on.
+
+<div align="center">
+
+## Acceleration {#TM:acceleration}
+
+</div>
+
+|Refname    |TM:acceleration                                                                                                                                                                                                                                                 |
+|:----------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|Label      |Acceleration                                                                                                                                                                                                                                                    |
+|Equation   |\\[\boldsymbol{a}\text{(}t\text{)}=\frac{\\,d\boldsymbol{v}\text{(}t\text{)}}{\\,dt}\\]                                                                                                                                                                         |
+|Description|<ul><li>\\(\boldsymbol{a}\text{(}t\text{)}\\) is the acceleration (\\(\frac{\text{m}}{\text{s}^{2}}\\))</li><li>\\(t\\) is the time (\\({\text{s}}\\))</li><li>\\(\boldsymbol{v}\text{(}t\text{)}\\) is the velocity (\\(\frac{\text{m}}{\text{s}}\\))</li></ul>|
+|Source     |[accelerationWiki](#accelerationWiki)                                                                                                                                                                                                                           |
+|RefBy      |                                                                                                                                                                                                                                                                |
+
+<div align="center">
+
+## Velocity {#TM:velocity}
+
+</div>
```

Comment:
Does this work in most Markdown renderers?

Reply:
The heading attributes (`{#id}`) are specific to mdBook, other Markdown renderers require an `id` attribute in the tag. Additionally, the definition headings are aligned to the center as tables in mdBook are centered. For other Markdown renderers aligning the heading to the center is not needed, as all elements are left-justified.

Reply:
Since we know that this is the GHFM flavour, this works. So, this comment can be dismissed. Thank you @BilalM04